modules/api/current.js

```diff
@@ -142,16 +145,30 @@ module.exports = {
         attributes: utils.createAttributes(feature.tags),
         codeRef: featureCodeRef,
       };
+
       const { tempId } = this.reportportal.startTestItem(suiteData, launchTempId, '');
       this.storage.setFeatureTempId(tempId);
-    } else {
-      const tempFeatureId = this.storage.getFeatureTempId();
-      this.reportportal.finishTestItem(tempFeatureId, {
+    } else if (isNewFeature) {
+      const previousFeatureTempId = this.storage.getFeatureTempId();
+      this.reportportal.finishTestItem(previousFeatureTempId, {
         endTime: this.reportportal.helpers.now(),
       });
+
+      this.storage.setCurrentFeatureUri(pickleFeatureUri);
+      const suiteData = {
+        name: feature.name,
+        startTime: this.reportportal.helpers.now(),
+        type: this.isScenarioBasedStatistics ? TEST_ITEM_TYPES.TEST : TEST_ITEM_TYPES.SUITE,
+        description: (feature.description || '').trim(),
+        attributes: utils.createAttributes(feature.tags),
+        codeRef: featureCodeRef,
+      };
```

Comment:
I see no difference between this object and another one [here](https://github.com/reportportal/agent-js-cucumber/pull/125/files#diff-e6c204e767b9a6e109df101103e321d0fee91883fc14e466767556cb210fada0R140).
Can we create `suiteData` once before conditions? I mean you can reorganize conditions and create one more f.e. `isNeedToStartFeature` and hold common things there. Further, I guess, it will be smoothly moved to the separate method.